Resources/Prototypes/_Goobstation/Entities/Objects/Devices/Electronics/misc_signallers.yml

```
@@ -0,0 +1,71 @@
+     sprite: _Goobstation/Objects/Devices/miscsignallers.rsi
+     state: voicesignaller
+   - type: Item
+     inhandVisuals:
+       left:
+       - state: voicesignaller_inhand_left
+       right:
+       - state: voicesignaller_inhand_right
+   - type: TriggerOnVoice
+   - type: PhysicalComposition
+     materialComposition:
+      Steel: 25
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 7 but found 6

(indentation)